docs/Configuration.md

```diff
@@ -102,12 +102,17 @@ This exporter supports several configuration options, passed via environment var
 

```

Comment:
I think we need to add a section here that explicitly states what "providers" we offer.